packages/cli/src/utils/deprecations.ts

```diff
@@ -0,0 +1,32 @@
+import { existsSync } from 'fs'
+import chalk from 'chalk'
+import { logger } from './logger'
+
+/**
+ * Checks for the presence of deprecated secret files and logs a warning if found.
+ *
+ * @param basePath The base path where to look for the secret files
+ */
+export function checkDeprecatedSecretFiles(basePath: string) {
+  const secretsFilePath = `${basePath}/secrets.hidden.json`
+  const vtexEnvFilePath = `${basePath}/vtex.env`
+
+  const secretsFileExists = existsSync(secretsFilePath)
+  const vtexEnvFileExists = existsSync(vtexEnvFilePath)
+
+  if (secretsFileExists || vtexEnvFileExists) {
+    const filesFound = [
+      secretsFileExists ? 'secrets.hidden.json' : null,
+      vtexEnvFileExists ? 'vtex.env' : null,
+    ]
+      .filter(Boolean)
+      .join(' and ')
+
+    logger.log(
```

Comment:
@emersonlaurentino log will only appear if the `DISCOVERY_DEBUG` env var is set. In case you want this to appear everywhere do a `logger.warn` instead.